### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by QuangVinh93**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,21 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BaseApp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= t "layouts.title" %></title>
+  <%= stylesheet_link_tag "application", media: "all",
+    "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: "data-turbolinks-track": true %>

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: <%= javascript_include_tag "application", "data-turbolinks-track": true %>

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="navbar navbar-inverse">
+  <div class="container">
+    <div class="navbar-header">
+      <%= link_to t("layouts.title"), root_path, class: "navbar-brand" %>
+    </div>
+    <ul class="nav navbar-nav">
+      <li><%= link_to t("layouts.home"), root_path %></li>
+      <li><%= link_to t("layouts.log_in"), "#"%></li>
```
Comment (tqtrung09) [inline app/views/layouts/_header.html.erb]: Have space before %>

**Comment by HUSTToanLH** [comment]
ok

**Comment by hitshootnum1** [comment]
ok

File: Gemfile [hunk 4]
```diff
@@ -1,47 +1,40 @@
...
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
-group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
+source "https://rubygems.org"
+
+gem "rails", "4.2.5"
```
Comment (dieunb) [inline Gemfile]: use latest rails version  `4.2.6`

File: Gemfile [hunk 5]
```diff
@@ -1,47 +1,40 @@
...
+gem "bcrypt", "3.1.7"
+gem "faker", "1.4.2"
+gem "carrierwave", "0.10.0"
+gem "mini_magick", "3.8.0"
+gem "fog", "1.36.0"
+gem "will_paginate", "3.0.7"
```
Comment (dieunb) [inline Gemfile]: use `kaminari` gem instead of this

File: Gemfile [hunk 6]
```diff
@@ -1,47 +1,40 @@
...
+gem "bootstrap-will_paginate", "0.0.10"
```
Comment (dieunb) [inline Gemfile]: remember remove this

**Pull request merged**